src/setupTests.ts

```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
```

Comment:
Let's create new directory called `tests/` nearby `src/` and put this file there (you can rename it as just `setup.ts`) as well.